superset/viz.py

```diff
@@ -491,6 +491,18 @@ def data(self):
     def get_csv(self):
         df = self.get_df()
         include_index = not isinstance(df.index, pd.RangeIndex)
+        if config.get("CSV_EXPORT_USE_VERBOSE_NAMES_AS_HEADERS"):
+            d = {
+                o.column_name: o.verbose_name
+                for o in self.datasource.columns
+                if hasattr(o, "verbose_name")
```

Comment:
I'm not sure how these objects usually look, but what if the attr is present, but is `None` or `''`? I would probably add a check to see that it's defined, aka. `if hasattr(o, "verbose_name") and o.verbose:` Do you agree? also, `o` could probably be called `column`.

Reply:
![image](https://user-images.githubusercontent.com/4054836/60972697-5d0c1300-a359-11e9-8d69-01f2f7fa7357.png)
Yes you are right, sometimes `verbose_name` would be `None`.